exercises/nucleotide-count/nucleotide_count_test.go

```diff
@@ -79,6 +83,10 @@ var histogramTests = []histogramTest{
 		Histogram{'A': 0, 'C': 0, 'T': 0, 'G': 8},
 	},
 	{
+		"GGXXGGGG", // skip mutation
```

Comment:
A new case I am a little unsure about! Made harder by the fact that `DNA` doesn't return an `error`. Should it? That (plus this change) might be for a separate PR though.

This is difficult for me to imagine, since x-common doesn' thave canonical-data.json for nucleotide-count. Have other tracks with nucleotide-count similar cases to this?